### PR TITLE
add focusrite-control-2:latest

### PR DESCRIPTION
**Pull request by JonathanXDR**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by JonathanXDR** [comment]
Hey guys,

Just a quick note, focusrite-control-2 is intended for users with 4th generation and newer Focusrite audio devices. This cask is distinct from the existing focusrite-control cask, which caters to earlier generation devices.

The focusrite-control-2 application is a ground-up rebuild designed specifically for newer hardware and is meant to replace the original "Focusrite Control" software for users with compatible devices. However, since it only works with 4th generation and newer devices, it does not supersede the original application for users with older hardware. Therefore, maintaining separate casks for each version ensures that all Focusrite users, regardless of their device generation, have access to the appropriate software.

File: Casks/f/focusrite-control-2.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  uninstall quit:      "com.focusrite.control",
+            launchctl: "com.focusrite.ControlServer",
+            delete:    [
+              "~/Applications/Focusrite Control 2.app",
+              "/Library/LaunchDaemons/com.focusrite.ControlServer.plist",
+            ]
+
+  zap trash: [
+    "~/Library/Application Support/Focusrite",
+    "~/Library/Caches/com.juce.locks",
```
Comment (bevanjkay) [inline Casks/f/focusrite-control-2.rb:23]: This line looks unrelated

File: Casks/f/focusrite-control-2.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  homepage "https://focusrite.com/en"
+
+  depends_on macos: ">= :mojave"
+
+  app "Focusrite Control 2.app"
+
+  uninstall quit:      "com.focusrite.control",
+            launchctl: "com.focusrite.ControlServer",
+            delete:    [
+              "~/Applications/Focusrite Control 2.app",
```
Comment (bevanjkay) [inline Casks/f/focusrite-control-2.rb:17]: This *should* happen automatically. Have you tried with this line?

File: Casks/f/focusrite-control-2.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+cask "focusrite-control-2" do
+  version "1.111.0.15742"
+  sha256 "1234"
```
Comment (bevanjkay) [inline Casks/f/focusrite-control-2.rb:3]: Update after next CI run, was taking a long time on my local connection.

File: Casks/f/focusrite-control-2.rb (hunk 1 above)
Comment (JonathanXDR) [reply Casks/f/focusrite-control-2.rb:28]: Yeah, I don't know either why this file has a completely different name. However, it does belong to the application. The original `focusrite-control` cask has it included as well.
Bildschirmfoto 2023-12-04 um 12 53 40

Comment (bevanjkay) [reply Casks/f/focusrite-control-2.rb:28]: Okay, that makes sense

**Comment by bevanjkay** [comment]
@JonathanXDR Thanks for submitting this. I was able to find a versioned `url` by finding url used by the in-app update mechanism that points to a Sparkle feed (now using for a `livecheck` stanza). Can you set [`auto_updates true` ](https://docs.brew.sh/Cask-Cookbook#optional-stanzas) and then it looks good to me.

**Review by bevanjkay — APPROVED** [review]
